docs/release/trg-9/trg-9-01.md

```diff
@@ -0,0 +1,76 @@
+---
+title: TRG 9.01 - UX consistency / Styleguide for UI
+sidebar_position: 1
+---
+
+| Status | Created       | Post-History                                             |
+|--------|---------------|----------------------------------------------------------|
+| Active | 22-Jul-2024   | Finalize TRG                                             |
+| Draft  | 25-April-2024 | Added new styleguide criteria - initial version          |
+| Draft  | 10-Jan-2024   | Transferred from Release Acceptance Criteria into TRG    |
+
+## Introduction
+
+The purpose of this guideline is to ensure that business applications with user interfaces adhere to the established UI/UX styleguide which is collected in the [Portal Shared Components](https://github.com/eclipse-tractusx/portal-shared-components.git). It outlines all the relevant criteria that must be checked when releasing a new product version. A crosscheck with every pull request is not just optional, but it is a "MUST" criteria to ensure the quality and consistency of the user interfaces of each product/component. This step is crucial in maintaining the overall standard and enhancing as well as standardizing the user experience.
+
+## Objective
+
+The objective is to ensure consistency, usability, and adherence to design standards across all user interfaces within the business application.
+
+## Criteria (Basic)
+
+### Styleguide Compliance
+
+- All user interface elements must match the predefined styles. This includes fonts, colors, button styles, input fields, and layout grids.
+  - [Storybook](https://eclipse-tractusx.github.io/portal-shared-components)
+- Icons and imagery should be consistent (note that images and icons must be open source compliant and license files are needed)
+  - [Image, Graphics & Diagrams](https://github.com/eclipse-tractusx/portal-assets/blob/v2.0.0/docs/developer/Technical%20Documentation/Others/Images%2C%20Graphics%20and%20Diagrams.md)
+
+### Responsiveness and Compatibility
+
+- The application must be fully responsive and provide a seamless experience across different devices and screen sizes.
+- The application must be tested with multiple browsers - in case any browsers are not supported, the same must be documented.
+
+### Accessibility
+
+- The application must meet the minimum accessibility standards (e.g., WCAG 2.1).
+- Optional: All user interface components must be navigable and usable with keyboard-only controls.
+- Optional: Screen reader compatibility must be tested and ensured.
+
+### User Experience
+
+- The application's user flow must align with the user experience outlined in the styleguide.
+- Interactive elements must provide feedback to the user, such as hover states, active states, and disabled states, as per the styleguide.
```

Comment:
Maybe a more clear checklist could help to know if you are compliant or not.

Reply:
there was already a previous comment suggesting that checklist is not the format for TRGs https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/865#pullrequestreview-2027713429, and I agree that that format would fit more to a pull request template.

Reply:
Ok to add it as PR template would be good also, like the "changelog" or "dependencies" thing, check if it complies with the guidelines. However I am not really sure how can you really mark if you "compliant" with the styleguidelines, its really substantial and depends from person to person...